teuthology/task/internal/start_minions.py

```diff
@@ -0,0 +1,27 @@
+import contextlib
+import logging
+
+from teuthology import misc
+from teuthology.orchestra import run
+
+
+log = logging.getLogger(__name__)
+
+
+@contextlib.contextmanager
+def start_minions(ctx, config):
+    """
+    start Salt Minions on all machines in the cluster
+    """
+    log.info('Starting salt-minion ...')
+
+    for rem in ctx.cluster.remotes.iterkeys():
+        log.debug("Deleting key of minion {}".format(rem.name))
+        misc.sh("salt-key -d {}".format(rem.name))
```

Comment:
Without having tested it, I think you want to add ```-y``` to the salt-key call. Otherwise it will want to interactively confirm the delete iirc.